code/game/objects/items/devices/PDA/PDA_types.dm

```diff
@@ -33,7 +32,9 @@
 /obj/item/pda/ai/pai
 	ttone = "assist"
 
-
+/obj/item/pda/ai/pai/Initialize()
```

Comment:
I think you wanted just `/obj/item/pda/ai/Initialize()`

Reply:
i did